junit-async/native/src/main/scala/scala/scalanative/junit/async/package.scala

```diff
@@ -0,0 +1,12 @@
+package scala.scalanative.junit
+
+import scala.concurrent.Future
+import scala.util.Try
+
+package object async {
+  type AsyncResult = Try[_]
+  def await(future: Future[_]): AsyncResult = {
+    scala.scalanative.runtime.loop()
+    future.value.get
```

Comment:
Since `AsyncResult` will be `Unit`, this should have an extra `.get` to unwrap the `Try` and potentially throw its exception:
```suggestion
    future.value.get.get
```